packages/sonataflow-deployment-webapp/README.md

```diff
@@ -39,7 +39,8 @@ To create a consumer app for local tests following the Quarkus Getting Started A
 
 ### Using a local webjar
 
-1.  Edit the `pom.xml` file and add the following plugin configuration:
+1.  From this project run `mvn install`
```

Comment:
Why this is necessary?

Reply:
`mvn install` builds the project and installs it in the local `.m2` repository.

Reply:
Yes, I know. But why is that necessary to be in the instruction?

Reply:
it's necessary to have it available in the `.m2` repo, to use it in the consumer app

Reply:
Got it, thanks!
Can you include this info in the step?
Something like:
```suggestion
1.  From this project run `mvn install` to have it available in the `.m2` repository so you can use it in your consumer app
```